docs/release-notes/release-notes-0.18.3.md

```diff
@@ -60,6 +60,10 @@ commitment when the channel was force closed.
 * We'll now always send [channel updates to our remote peer for open
   channels](https://github.com/lightningnetwork/lnd/pull/8963).
 
+* [A bug has been fixed that could cause invalid channel
```

Comment:
Cool think we need to do a new rc?

Reply:
Yeah, I think we have another PR or two incoming to trigger a new rc.